stories/atoms/components/textField/TextField.stories.mdx

```diff
@@ -139,23 +147,49 @@ In addition, the `helperText` property can be set to display an error message be
   </Story>
 </Canvas>
 
-## Reacting to changes
+## Controlled TextField
 
-The `TextField` component provides an`onChange` property called when the input value changes.
+When the component is controlled, the parent passes a value to the `TextField` which displays it.  
+Parent has its own state in order to control the value which is passed, and it also specify an `onChange` handler property which will be called when user updates the value.  
+Then, everytime the input value changes, current state is updated and a log message displays the content of the value.
 
-##### **`MyComponent.tsx`**
+<Canvas isColumn>
+  <Story name="Controlled">
+    {args => {
+      const [value, setValue] = useState('I am controlled: value is passed by my parent')
+      const handleChange = event => {
+        setValue(event.target.value)
+        console.log(`My new value is : ${event.target.value}`)
```

Comment:
`console.log` remaining

Reply:
The console.log was only for the example 😉, I just specified it in the story to make it clearer, thanks!